APTITUDE/APTITUDETEST.Core/ViewModels/UserDetailsVM.cs

```diff
@@ -8,22 +8,29 @@ public class UserDetailsVM
         public string? FatherName { get; set; }
         public string Email { get; set; }
         public long? PhoneNumber { get; set; }
-        public int Level { get; set; }
+
         public DateTime? DateOfBirth { get; set; }
-        public string? PermanentAddress { get; set; }
+        public string? PermanentAddress1 { get; set; }
+        public string? PermanentAddress2 { get; set; }
+        public int? Pincode { get; set; }
+        public int? City { get; set; }
```

Comment:
Is it required?

Reply:
Candidate Details page has these fields.